guides/common/modules/proc_viewing-booted-container-images.adoc

```diff
@@ -0,0 +1,18 @@
+[id="Viewing_booted_container_images_{context}"]
+= Viewing booted container images
+
+These actions streamline the management of image mode hosts, providing flexibility and control over host operations through {Project}.
+
+.Prerequisites
+* *Pull-Mode Remote Execution*: For pull-mode remote execution users, the `foreman_ygg_worker` package must be included in the image mode host's container image.
+
+.Procedure
+. Navigate to *Content* > *Booted Container Images* to view all images used by image mode hosts.
+This page displays the spread of digests under specific image paths.
+More digests under an image path indicate greater drift in the host ecosystem, ideally minimized by ensuring all hosts run the most up-to-date image version.
+. Click the host count to navigate to the *All Hosts* page, which identifies the hosts associated with each specific image and displays the number of hosts using each digest.
+. Select a host and on the *Details* tab, use the image mode *Details* card to see the current `bootc` status.
+. Click the link to the *Bootc Action - Script Default* remote execution job in this card to perform a Bootc action on the host.
```

Comment:
While it's true that that's where it links to, that is not the text of the link.
![image](https://github.com/user-attachments/assets/5c2cd166-1d50-4462-bd63-2430648303f0)